plugins/emoji/README.md

```diff
@@ -25,7 +25,30 @@ On additional key presses it allows for fuzzy search on the labels of each emoji
 
 ## Installation instructions
 
-Install [em-keyboard](https://github.com/hugovk/em-keyboard) via `pip3`:
+### Requisites
+
+#### Exec requisites
+
+- `xclip`: you must have installed xclip; ensure that xlip path is in PATH environment variable.
+
+_Example for Ubuntu_:
+
+```bash
+sudo apt-get install xclip
+which xclip  # i.e. /usr/bin/xclip
+export PATH="$PATH:/usr/bin/xclip"
```

Comment:
For this to work you'd have to write

```
export PATH="$PATH:/usr/bin/" # <-- directory that xclip is in
```

But then again `/usr/bin` is always in your PATH on Ubuntu (and most other distros)

Also, if `which xclip` does find the executable, then it's most likely already in your PATH (see `man xclip` for more).

I'd skip the bash snippet in this case and just stick to:

> `xclip`: you must have installed xclip; ensure that xlip path is in PATH environment variable.`

Users will probably know how to modify PATH if required

Reply:
You're right, I'll change that